app/assets/stylesheets/application_drafts/edit.scss

```diff
@@ -23,13 +23,14 @@ html {
 
 #flex_wrapper{
   display: flex;
+  padding-bottom: 20px;
   .form_column{
     flex:3;
     //list-style-type:none;
     .header_row{
       padding: .5em;
       display: flex;
-      height: 2em;
+      //height: 2em;
```

Comment:
I agree, I think I was messing with various values and accidentally left it in. Must've missed this when I was scanning for comments.